src/portable/create.rs

```diff
@@ -190,10 +190,54 @@ fn create_cloud(cmd: &Create, org_slug: &str, name: &str, client: &cloud::client
     )?;
 
     let server_ver = cloud::versions::get_version(&query, client)?;
+    let tier = if cmd.free_tier {
+        String::from("Free")
+    } else {
+        String::from("Pro")
+    };
+
+    let compute_size = cmd.compute_size.unwrap_or(1);
+    let storage_size = cmd.storage_size.unwrap_or(20);
+
+    let req_resources = match cmd.free_tier {
+        true => {
+            None
+        },
+        false => {
+            Some(vec![
+                cloud::ops::CloudInstanceResourceRequest{
+                    name: "compute".to_string(),
+                    value: compute_size,
+                },
+                cloud::ops::CloudInstanceResourceRequest{
+                    name: "storage".to_string(),
+                    value: storage_size,
+                },
+            ])
+        },
+    };
+
+    let resources_display = if req_resources.is_some() {
```

Comment:
we might want to display this even for free-tier? that way people know what they're getting in terms of compute / storage, and whets their appetite for "ooh, I want more storage..." and converting them to paid customers

Reply:
Done